test/bigml/test/api/source.clj

```diff
@@ -8,11 +8,13 @@
   (:use clojure.test))
 
 (defn- create-and-test [artifact & params]
-  (let [initial (api/get-final (apply source/create artifact params))
+  (let [{:keys [username api_key dev_mode] :as sticky-pars} params
```

Comment:
You can use the `:as` to bind params as a hash-map in the fn definition, which makes using `select-keys` a little less messy. Like so:

```clojure
(defn bar [a & b]
  (println "Param seq: " b))

(defn foo [a & {:as b}]
  (let [c (flatten (seq (select-keys b [:x :y])))]
    (apply bar a c)))

;; user> (foo 0 :x 1 :y 2 :z 3)
;; Param seq:  (:y 2 :x 1)
```